ihmc-communication/src/main/java/us/ihmc/communication/crdt/RequestConfirmFreezable.java

```diff
@@ -114,9 +121,9 @@ public void toMessage(ConfirmableRequestMessage message)
 
       if (needToSendRequest.poll())
       {
-         long requestNumber = nextRequestID.incrementAndGet();
+         long requestNumber = nextRequestID.getAndIncrement();
          unconfirmedRequests.add(requestNumber);
-         requestTimeouts.put(requestNumber, updateNumberToUnfreeze);
+         requestTimeouts.put(requestNumber, crdtInfo.getUpdateNumber() + crdtInfo.getMaxFreezeDuration());
```

Comment:
Was straight up wrong and allowed there to be an "unconfirmed" error when messages happen in weird orders and a confirmation didn't even have a chance to arrive yet